align/pnr/build_pnr_model.py

```diff
@@ -189,7 +143,11 @@ def PnRdatabase( path, topcell, vname, lefname, mapname, drname):
     DB.ReadPDKJSON( path + '/' + drname)
 
     _ReadLEF( DB, path, lefname)
-    DB.gdsData = _ReadMap( path, mapname)
+    DB.gdsData, DB.gdsData2 = _ReadMap( path, mapname)
```

Comment:
@854768750 Here is were the Python map get copied to the C++ map. (and the printed out so we can see it is doing the right thing.)